06_puppy/pfannerj/storer_test.go

```diff
@@ -0,0 +1,180 @@
+package main
+
+import (
+	"testing"
+
+	tassert "github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type storerImpl int
+
+const (
+	syncImpl storerImpl = iota // 0
```

Comment:
I don't think you need to put what the value is in the comment, but not a big deal if you leave it.

Reply:
Please fix.

Reply:
Fixed